sdk/quantum/Azure.Quantum.Jobs/src/Azure.Quantum.Jobs.csproj

```diff
@@ -24,6 +26,10 @@
     <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)PageResponseEnumerator.cs" />
+  </ItemGroup>
+  
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
 
 </Project>
```

Comment:
Making a comment here (since github won't let me comment on moved files) to edit tests.yml and ci.yml to change `template` to `quantum`, `AzureTemplate` to `AzureQuantumJobs`, and `Azure.Template` to `Azure.Quantum.Jobs`.  You'll want to fix this before merging otherwise the build pipelines will get kind of confused and start treating your project as part of the Template.